.github/workflows/alert-docs.yml

```diff
@@ -29,11 +28,12 @@ jobs:
         # Run the ZAP script
         docker run -v $(pwd):/zap/wrk/:rw --user root -t owasp/zap2docker-live ./zap.sh -addoninstallall -silent -script /zap/wrk/zap-admin/scripts/generate_alert_pages.js -cmd
         cd zaproxy-website
+        git checkout -b update-alerts
```

Comment:
Is this going to work after the first run? (When the branch already exists)

Reply:
Its a new VM (or whatever it is) each time, isnt it? So I _think_ it will be fine...

Reply:
Yup, you're right